### PR TITLE
Viewschedule1.32016

### PR DESCRIPTION
**Pull request by mjkkirschner**
### Purpose

This PR cherry picks multiple PRs from Ksobons view schedule work to 1.3 branch.

@QilongTang please take a look...

### CODE REVIEW

**Comment by mjkkirschner** [comment]
@QilongTang this builds now - PTAL.

File: src/Libraries/RevitNodes/Elements/FaceWall.cs [hunk 1]
```diff
@@ -119,7 +119,7 @@ private void InternalSetFaceWall(Autodesk.Revit.DB.FaceWall wall)
         /// <param name="wallType"></param>
         /// <param name="surface"></param>
         /// <returns></returns>
-        /// Note: This moved to the Wall element
+        /// Note: This is wrapped in Wall. FaceWalls will be created there.
```
Comment (QilongTang) [inline src/Libraries/RevitNodes/Elements/FaceWall.cs]: Thanks, this was missing and should be cherry picked

File: src/Libraries/RevitNodes/Elements/FillPatternElement.cs [hunk 2]
```diff
@@ -0,0 +1,114 @@
+﻿using Autodesk.DesignScript.Runtime;
```
Comment (QilongTang) [inline src/Libraries/RevitNodes/Elements/FillPatternElement.cs]: Fill pattern is not part of the plan but does not do harm I guess.. Just more testing effort~

Comment (QilongTang) [inline src/Libraries/RevitNodes/Elements/FillPatternElement.cs]: Sorry part of the viewschedule export, should be included

File: src/Libraries/RevitNodes/Properties/Resources.resx [hunk 3]
```diff
@@ -372,4 +375,79 @@
...
+  <data name="NoSharedParameterFileFound" xml:space="preserve">
+    <value>No shared parameter file found.</value>
+  </data>
+  <data name="ParameterGroupNotFound" xml:space="preserve">
+    <value>Invalid parameter group.</value>
+  </data>
+  <data name="ParameterTypeNotFound" xml:space="preserve">
+    <value>Invalid parameter type.</value>
+  </data>
+  <data name="CurveIsNotPlanar" xml:space="preserve">
```
Comment (QilongTang) [inline src/Libraries/RevitNodes/Properties/Resources.resx]: Some of these looks irrelevant but does not harm adding them

**Comment by QilongTang** [comment]
@mjkkirschner Looks good, this has some extra resource string but that does no harm
